index.bs

```diff
@@ -3283,34 +3283,28 @@ size of 1024, it attempts to fill the developer-supplied buffer, allowing full c
       },
 
       pull(controller) {
-        if (controller.byobRequest) {
-          const v = controller.byobRequest.view;
-
-          return fs.read(fd, v.buffer, v.byteOffset, v.byteLength, position).then(bytesRead => {
-            if (bytesRead === 0) {
-              return fs.close(fd).then(() => controller.close());
-            } else {
-              position += bytesRead;
-              controller.byobRequest.respond(bytesRead);
-            }
-          });
-        } else {
-          // The consumer is using the default reader.
-          const buffer = new ArrayBuffer(DEFAULT_CHUNK_SIZE);
-          return fs.read(fd, buffer, 0, DEFAULT_CHUNK_SIZE, position).then(bytesRead => {
-            if (bytesRead === 0) {
-              return fs.close(fd).then(() => controller.close());
-            } else {
-              position += bytesRead;
-              controller.enqueue(new Uint8Array(buffer, 0, bytesRead));
-            }
-          });
-        }
+        // Even when the consumer is using the default reader, the auto allocate feature allocate a buffer and pass it
```

Comment:
Yeah, I'm just following the 120 col rule for all. I can live and like shorter wrapping. How much limit shall we use?